### PR TITLE
Icons Added in language menu

### LINKED ISSUE #3565: New UI for the Language Select Menu which would include 'add' and 'close' icons
**Actual Behaviour**

Currently when a user clicks on a language option it just highlights itself. It will be great if we can provide the menu with dynamic icons.

**Expected Behaviour**

We can add some icons which changes accordingly. A 'x' icon will be displayed in front of a selected language option and similarly a '+' in front of unselected options which would change dynamically upon toggling the selections

Screenshot (123)

**Would you like to work on the issue?**

Yes, I would love to work on the issue

### PR DESCRIPTION
**Pull request by Akhil-Kashyap**
Fixes #3565 

Changes: Respective Icons were added to the Menu which changes accordingly.

**Before**
Screenshot (123)

**After**
after_pull

### CODE REVIEW

File: src/components/NavigationBar/index.js [hunk 1]
```diff
@@ -103,7 +105,7 @@ const SusiLogo = styled.img`
     props.marginRight &&
     css`
       margin-right: ${props => props.marginRight + 'px'};
-    `}
+    `};
```
Comment (anishagg17) [inline src/components/NavigationBar/index.js:108]: prettier issue

**Comment by Akhil-Kashyap** [comment]
> Better UX would be similar to the one shown in the 2nd example in this link - https://material-ui.com/components/selects/#multiple-select, which uses a checkbox.



> Better UX would be similar to the one shown in the 2nd example in this link - https://material-ui.com/components/selects/#multiple-select, which uses a checkbox.

Sure, I'll look forward to implement this UI.

**Comment by Akhil-Kashyap** [comment]
Screenshot (126)

I have implemented the following UI . Please have a look @akshatnitd

**Comment by Akhil-Kashyap** [comment]
Done with the changes.
Screenshot (127)

**Pull request closed without merging**